packages/oauth/src/openshift/osoauth-service.ts

```diff
@@ -283,14 +283,13 @@ export class OSOAuthService implements OAuthProtoService {
   private doLogout(config: OpenShiftOAuthConfig): void {
     this.fetchUnregister?.()
 
-    const currentURI = new URL(window.location.href)
     // The following request returns 403 when delegated authentication with an
     // OAuthClient is used, as possible scopes do not grant permissions to access the OAuth API:
     // See https://github.com/openshift/origin/issues/7011
     //
     // So little point in trying to delete the token. Lets do in client-side only
     //
-    forceRelogin(currentURI, config)
+    forceRelogin(new URL(window.origin), config)
```

Comment:
Does `window.online` really return the intended URL?  It doens't seem to include `/<path>` part. Isn't it better to use `hawtio.getBasePath()`, which is the standard Hawtio API to get the console's base path?

Reply:
`hawtio.getBasePath()` can return undefined and I thought it will return only `/online`

Reply:
You are right, so you should use the combination like:
https://github.com/hawtio/hawtio-next/blob/7acdf61d5f4839f26cd33f35b81560fcb74aa3ae/packages/hawtio/src/plugins/shared/jolokia-service.ts#L421-L422

Are you sure that `window.origin` returns `/online`?  I couldn't find resources that assure so.
https://developer.mozilla.org/en-US/docs/Web/API/origin
https://stackoverflow.com/questions/55451493/what-is-window-origin
To me it looks like it returns something like `https://<cluster-hostname>:<port>`, just like `Origin:` http header.

Reply:
I can only find: https://html.spec.whatwg.org/multipage/browsers.html#origin

Reply:
`window.location.origin` seems to be safer: https://developer.mozilla.org/en-US/docs/Web/API/Location/origin in combination with `hawtio.getBasePath()`